apps/pie-docs/src/assets/styles/components/_tabs.scss

```diff
@@ -5,7 +5,8 @@ $tabs-link-padding:                var(--dt-spacing-d);
 $max-scroll-height:                var(--dt-spacing-d);
 
 .c-tabs {
-    z-index: f.zIndex(mid);
+    position: relative;
+    z-index: 1;
```

Comment:
Do you think we should have a base zIndex variable for this use-case?

Reply:
that's a good idea, do you have something in mind for names?

Reply:
Maybe `--dt-z-index-base: 1;`